tests/cloudpickle_test.py

```diff
@@ -652,6 +652,15 @@ def __init__(self, x):
 
         self.assertEqual(set(weakset), set([depickled1, depickled2]))
 
+    def test_namedtuple(self):
```

Comment:
What do you think about also testing that we can pickle/unpickle `MyTuple` (the class itself).